Proofreading-OV-Runtime (#11658)
openvinotoolkit/openvino: tools/pot/docs/BestPractices.md

## Introduction
The [Default Quantization](@ref pot_default_quantization_usage) of the Post-training Optimization Tool (POT) is 

> the fastest and easiest way to get a quantized model. It requires only some unannotated representative dataset to be provided in most cases. Therefore, it is recommended to use it as a starting point when it comes to model optimization. However, it can lead to significant accuracy deviation in some cases. This article purpouse is to provide tips to address this issue.

Author: ```suggestion
the fastest and easiest way to get a quantized model. It requires only some unannotated representative dataset to be provided in most cases. Therefore, it is recommended to use it as a starting point when it comes to model optimization. However, it can lead to significant accuracy deviation in some cases. The purpose of this article is to provide tips to address this issue.
```

```python
{
    "name": "DefaultQuantization", # Optimization algorithm name
    "params": {
        "preset": "performance", # Preset [performance, mixed] which controls 
                                 # the quantization scheme. For the CPU: 
                                 # performance - symmetric quantization  of weights and activations.
                                 # mixed - symmetric weights and asymmetric activations.
        "stat_subset_size": 300  # Size of subset to calculate activations statistics that can be used
                                 # for quantization parameters calculation.
    }
}
```

There are two alternatives in case of substantial accuracy degradation after applying this method:
1.  Hyperparameters tuning.
2.  AccuracyAwareQuantization algorithm.

### Tuning Hyperparameters of the Default Quantization
The Default Quantization algorithm provides multiple hyperparameters which can be used in order to improve accuracy results for the fully-quantized model. 
Below is a list of best practices that can be applied to improve accuracy without a substantial performance reduction with respect to default settings:

> 1.  The first recommended option is to change the `preset` from `performance` to `mixed`. This enables asymmetric quantization of 

Author: ```suggestion
1.  The first recommended option is to change the *preset* from `performance` to `mixed`. This enables asymmetric quantization of 
```

> 1.  The first recommended option is to change the *preset* from `performance` to `mixed`. This enables asymmetric quantization of 

Reviewer: I think it was to highlight that "preset" value is the one to change. But maybe instead of using italic, let's change it to that:

```suggestion
1.  The first recommended option is to change the **preset** from `performance` to `mixed`. This enables asymmetric quantization of 
```

> 1.  The first recommended option is to change the *preset* from `performance` to `mixed`. This enables asymmetric quantization of 

Reviewer 2: I would be cautious, the preset here may be a parameter, in which case the code formatting would fit.

activations and can be helpful for models with non-ReLU activation functions, for example, YOLO, EfficientNet, etc.

> 2.  The second option is `use_fast_bias`. Setting this option to `false` enables a different bias correction method which is more accurate, in general,

Author: ```suggestion
2.  The second option is the `use_fast_bias`. Setting this option to `false` enables a different bias correction method which is more accurate, in general,
```

and applied after model quantization as a part of the Default Quantization algorithm.

>    > **NOTE**: Changing this option can substantially increase quantization time in the POT tool.

Author: ```suggestion
   > **NOTE**: Changing this option can substantially increase quantization time in POT tool.
```

> 3.  Another important option is `range_estimator`. It defines how to calculate the minimum and maximum of quantization range for weights and activations.

Author: ```suggestion
3.  Another important option is the `range_estimator`. It defines how to calculate the minimum and maximum of quantization range for weights and activations.
```

Find the possible options and their description in the `configs/default_quantization_spec.json` file in the POT directory.

4.  The next option is `stat_subset_size`. It controls the size of the calibration dataset used by POT to collect statistics for quantization parameters initialization.
It is assumed that this dataset should contain a sufficient number of representative samples. Hence, varying this parameter may affect accuracy (higher is better). 

> However, it shows that 300 samples are sufficient to get representative statistics in most cases.

Author: ```suggestion
However, it proves that 300 samples are sufficient to get representative statistics in most cases.
```

> 5.  The last option is `ignored_scope`. It allows excluding some layers from the quantization process, for example, their inputs will not be quantized. It may be helpful for some patterns, which are known in advance that they drop accuracy when executing in low-precision.

Author: ```suggestion
5.  The last option is the `ignored_scope`. It allows excluding some layers from the quantization process, for example, their inputs will not be quantized. It may be helpful for some patterns, which are known in advance, that they drop accuracy when executing in low-precision.
```

> For example, `DetectionOutput` layer of SSD model expressed as a subgraph should not be quantized to preserve the accuracy of Object Detection models.

Author: ```suggestion
For example, the `DetectionOutput` layer of SSD model expressed as a subgraph should not be quantized to preserve the accuracy of Object Detection models.
```

One of the sources for the ignored scope can be the Accuracy-aware algorithm which can revert layers back to the original precision (see the details below).

## Accuracy-aware Quantization

> In situation when the steps above do not lead to the accurate quantized model you may use the so-called [Accuracy-aware Quantization](@ref pot_accuracyaware_usage) algorithm which leads to mixed-precision models. 

Author: ```suggestion
When the steps above do not lead to the accurate quantized model, you may use the so-called [Accuracy-aware Quantization](@ref pot_accuracyaware_usage) algorithm, which leads to mixed-precision models. 
```

Since the Accuracy-aware Quantization calls the Default Quantization at the first step it means that all the parameters of the latter one are also valid and can be applied to the accuracy-aware scenario.

> **NOTE**: In general, possible speedup after applying the Accuracy-aware Quantization algorithm is less than after the Default Quantization when the model gets fully quantized.

### Reducing the performance gap of Accuracy-aware Quantization

> To improve model performance after Accuracy-aware Quantization try the `"tune_hyperparams"` setting and set it to `True`. It will enable searching for optimal quantization parameters before reverting layers to the "backup" precision. Yet, remember that this can increase the overall quantization time.

Author: ```suggestion
To improve model performance after Accuracy-aware Quantization, try the `"tune_hyperparams"` setting and set it to `True`. It will enable searching for optimal quantization parameters before reverting layers to the "backup" precision. Yet, remember that this can increase the overall quantization time.
```
